lib/originals-helper.js

```diff
@@ -56,6 +56,61 @@ function getModuleNameForType(type) {
   }
 }
 
+/**
+ * Traverses the prototype chain to find and return the module
+ * from which a global property originated
+ *
+ * @param {Node} node - a ts-morph AST node
+ * @param {string} property - the property for which to find the module
+ *
+ * @return {string} the get-originals module name corresponding to the
+ * type. (This is sometimes different from the TypeScript type name.)
+ */
+function getModuleNameForPropertyOfNode(node, property) {
+  const type = node.getType();
+
+  // Need to use BaseTypeOfLiteralType to catch the type of primitives
+  switch (type.getBaseTypeOfLiteralType().getText()) {
+    case 'string': {
+      return 'String';
+    }
+    case 'number': {
+      return 'Number';
+    }
+    case 'boolean': {
+      return 'Boolean';
+    }
+  }
+
+  // Find the deepest reference to the property in the prototype chain
+  let inspectType = type;
+  let latestFound;
+  while (inspectType !== undefined) {
+    if (inspectType.getProperty(property) !== undefined) {
+      latestFound = inspectType;
+    } else {
+      break;
+    }
+
+    inspectType = inspectType.getSymbol()
+        .getDeclarations()[0].getBaseTypes()[0];
```

Comment:
Add a comment explaining that the 0th base type is the real prototype chain base type, and others are mixins. Maybe with an example.